serverless-plugin/index.js

```diff
@@ -303,11 +301,12 @@ class ServerlessPlugin {
             const stackInstallRoot = this.runStackOutput(
                 directory,
                 [
+                    '--docker',
```

Comment:
There's already a mechanism for disabling Docker (search the file for `localRun` and `docker.skip`), please use that instead.

Reply:
For some reason, it always skips it, that's why I had to manually add it

Reply:
Are you talking about the integration tests? They also test "disable Docker completely" mode, not sure how that plays with the new `invoke local`.

Reply:
The new `invoke local` relies on always running Docker, so it won't work with that.

Still, I'm talking about running `sls invoke local` as it is. With the `--docker` flag, or without it, the result is the same one, the wrong executable gets copied (the one that is not built with Docker).